recipes/xsimd/all/conanfile.py

```diff
@@ -75,6 +76,7 @@ def _module_file_rel_path(self):
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "xsimd")
         self.cpp_info.set_property("cmake_target_name", "xsimd")
+        self.cpp_info.set_property("cmake_target_namespace", "xsimd")
```

Comment:
😕 Upstream does not set a namespace https://github.com/xtensor-stack/xsimd/blob/135c19a1286ea512db4316cf92f72266ed71b7a3/CMakeLists.txt#L159

```suggestion
```